bfsbverify

```diff
@@ -834,15 +833,12 @@ else
         $ddratedmem
     
     if [ "$version" = "2" ]; then
-        # Verify Ddr5SnpsHash
-        # The Ddr5SnpsHash is expected in
         # certificate extension.
         #   OID: 1.3.6.1.4.1.33049.2100.208 
         #
         verify_bl_digest \
             "1.3.6.1.4.1.33049.2100.208" \
             $ddrcert \
-            $ddr5snps
```

Comment:
Have you tested/verified these changes?

Reply:
I suggest we keep ddr5snps for backward compatibility with older BFB files.
Let's test whether ddr5snps is present before:

``
if [ "$version" = "2" ] && [ -f "$ddr5snps" ]; then
        # Verify Ddr5SnpsHash
        # The Ddr5SnpsHash is expected in
        # certificate extension.
        #   OID: 1.3.6.1.4.1.33049.2100.208 
        #
        verify_bl_digest \
            "1.3.6.1.4.1.33049.2100.208" \
            $ddrcert \
            $ddr5snps

fi
``